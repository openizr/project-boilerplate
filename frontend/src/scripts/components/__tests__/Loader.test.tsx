/**
 * @jest-environment jsdom
 */

import React from 'react';
import Loader from 'scripts/components/Loader';
import { render } from '@testing-library/react';

describe('components/Loader', () => {
  beforeEach(() => {
    jest.clearAllMocks();
  });

  test('renders correctly - basic', () => {
    const { container } = render(<Loader />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
