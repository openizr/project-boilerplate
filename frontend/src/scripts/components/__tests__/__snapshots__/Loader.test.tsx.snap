// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Loader renders correctly - basic 1`] = `
<div>
  <div>
    Loading...
  </div>
</div>
`;
